server/applicationset/applicationset.go

```diff
@@ -57,11 +54,10 @@ func NewServer(
 	db db.ArgoDB,
 	kubeclientset kubernetes.Interface,
 	enf *rbac.Enforcer,
-	cache *servercache.Cache,
 	appclientset appclientset.Interface,
 	appLister applisters.ApplicationLister,
```

Comment:
I see that `appLister` was removed from the struct. Does it still need to be in the function params?

Reply:
nice catch!